WARNING  selenium.webdriver.common.selenium_manager:selenium_manager.py:149 Error sending stats to Plausible: error sending request for url (https://plausible.io/api/event): operation timed out
INFO     Test_user_login:test_user_login_params.py:19 Testcase 'test_login_with_params' Started
INFO     Test_user_login:test_user_login_params.py:20 Opening the Browser
INFO     Test_user_login:test_user_login_params.py:22 Assigning Variable to Login Page
INFO     Test_user_login:test_user_login_params.py:24 Entering Email from Params -->admin@yourstore.com
INFO     Test_user_login:test_user_login_params.py:26 Entering Password from Params -->admin
INFO     Test_user_login:test_user_login_params.py:28 Clicking on Login Button
INFO     Test_user_login:test_user_login_params.py:30 Updating List
INFO     Test_user_login:test_user_login_params.py:33 Entering into 'outer if block'
INFO     Test_user_login:test_user_login_params.py:35 Entered into 'inner if block' of 'outer if block' -->pass
INFO     Test_user_login:test_user_login_params.py:36 Appending the list
INFO     Test_user_login:test_user_login_params.py:38 Clicking on logout Button